examples/OneOpenAir/OneOpenAir.ino

```diff
@@ -861,7 +864,9 @@ static void configUpdateHandle() {
       doOta = true;
       Serial.println("First OTA");
     } else {
-      if ((measurements.bootCount - measurements.otaBootCount) >= 30) {
+      /** Only check for update each 2h*/
+      const float otaBootCount = 120.0f / (SERVER_SYNC_INTERVAL / 60000.0f);
```

Comment:
We need to be more frequent than every 2h, propose to make it 1h to be on the safe side.